financial/simple_moving_average.py

```diff
@@ -0,0 +1,59 @@
+"""
+Calculate the Simple Moving Average (SMA) for a time series data.
+https://en.wikipedia.org/wiki/Moving_average
```

Comment:
```suggestion
The Simple Moving Average (SMA) is a statistical calculation used to analyze data points
by creating a constantly updated average price over a specific time period. In finance, SMA is 
often used in time series analysis to smooth out price data and identify trends.

Reference: https://en.wikipedia.org/wiki/Moving_average
```

The more in-depth explanation can go here